tfhe/benches/integer/signed_bench.rs

```diff
@@ -792,6 +792,7 @@ fn main() {
     match env::var("__TFHE_RS_BENCH_OP_FLAVOR") {
         Ok(val) => {
             match val.to_lowercase().as_str() {
+                "default" => default_ops(),
```

Comment:
this only has the signed if then else, are we sure that's what we expect ?

Reply:
It's that or this operations is never run.

Reply:
I mean is it really what we expect for the signed benchmarks compared to unsigned ones ? i.e. we will not get a value for the signed add parallelized e.g.

Reply:
Won't we have default operations in the same fashion as unsigned ones ?

Reply:
check on your machine but I did some quick benches a few days back and was missing basically all ops :/

Reply:
I'll check tomorrow on the m6i if I still have the patch laying around

Reply:
I've added the missing operations, thanks.